src/imanagement_api.ts

```diff
@@ -467,6 +479,19 @@ export interface IManagementApi {
                                processModelId: string,
                                flowNodeId: string): Promise<Array<TokenHistoryEntry>>;
 
+  /**
+   * Gets the token histories for a given CorrelationId and ProcessModelId.
+   *
+   * @async
+   * @param identity       The requesting users identity.
+   * @param correlationId  The ID of the Correlation.
+   * @param processModelId The ID of the ProcessModel.
+   * @returns              A list of discovered active tokens for the given
```

Comment:
```suggestion
   * @returns              A list of discovered tokens for the given
```

You don't just query active tokens here, but all tokens.